client/transactions/filters/test/__snapshots__/index.tsx.snap

```diff
@@ -5,13 +5,18 @@ HTMLOptionsCollection [
   <option
     value="online"
   >
-    Online
+    Online Store
   </option>,
   <option
     value="in_person"
   >
     In-Person
   </option>,
+  <option
+    value="in_person_pos"
+  >
+    In-Person (POS)
+  </option>,
```

Comment:
My understanding from the P2 discussion is that we would have two broad categories:

- `Online store`
- `In-person`

Each category would have more specific variations, like child terms in a taxonomy. The variations would be tracked in custom metadata/attribute. For example:

- `Online store`
  - `credit card`
  - `paypal`
  - `afterpay`
  - etc
- `In-person`
  - woo iOS app IPP
  - woo / stripe card-reader hardware
  - etc

So … I wasn't expecting two separate categories for `In-person`. I think that's confusing, and doesn't scale – we don't have separate top-level categories for `Online store`, even though it covers lots of variations.

Reply:
I agreed with the sentiment here, but actually this dropdown filter option has been missed this PR was merged https://github.com/Automattic/woocommerce-payments/pull/9802

So it's a compliment for that PR.